src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -682,14 +682,6 @@ public static boolean saplingGrowTree(IWorld world, Random rand, BlockPos pos)
         return event.getResult() != Result.DENY;
     }
 
-    public static void fireChunkWatch(boolean watch, ServerPlayerEntity entity, ChunkPos chunkpos, ServerWorld world)
```

Comment:
This is a breaking change as the method is public. Keep the method, but deprecate it and add a todo comment to remove the method in 1.15 if you no longer use it.

Reply:
Fixed.